main.go

```diff
@@ -281,7 +285,9 @@ func getRouter(logger *zap.Logger, config *Config, indexer Indexer) (*mux.Router
 	router.HandleFunc(signaturesRouterPath, signaturesHandler)
 	router.HandleFunc(packageIndexRouterPath, packageIndexHandler)
 	router.HandleFunc(staticRouterPath, staticHandler)
+	router.Handle("/metrics", promhttp.Handler())
```

Comment:
For production environments we probably want to expose this endpoint in other host/port, so metrics are not publicly available. We may need to add a flag for this.